cmd/create/cluster.go

```diff
@@ -151,6 +156,10 @@ func NewClusterCmd(version string, tracker *analytics.Tracker) *cobra.Command {
 				return fmt.Errorf("error while validating dependencies: %w", err)
 			}
 
+			if phase == "" || skipPhase == "distribution" {
+				vpnAutoConnect = true
+			}
```

Comment:
Are you sure we want this behavior here? wouldn't it be less surprising if we left the choice to be controlled by the `vpn-auto-connect` flag alone?

Reply:
depends if we want to force the user to set vpnAutoConnect when doing the phase == "" (all phases) and there's a VPN in the furyctl infra vpc section, if the user forgets this flag then the kubernetes phase fails badly with the only option to delete sgs,eks,... via other tools